src/scss/main.scss

```diff
@@ -0,0 +1,93 @@
+@import '~bootstrap/scss/bootstrap';
+// Variables
+$purple-color: #a64fae;
+body {
+  font-family: 'poppins', sans-serif;
+  background-color: grey;
+  background-color: #ffff;
+}
+.btn-flat {
+  background-color: #55dd55;
+  color: #000;
+}
+.colcenter {
+  padding: 1rem;
+  text-align: center;
+}
+button.btn-form {
```

Comment:
Move this inside the scss/styles.scss file, or better yet: create separate scss files inside the corresponding component folders.